docs/pages/get-started/react.mdx

```diff
@@ -134,26 +140,28 @@ We can re-export this from `liveblocks.config.js`, and because we’re using
 suspense in this example, we’ll export all our hooks from the `suspense`
 property.
 
-```jsx file="liveblocks.config.js" highlight="11"
+```ts file="liveblocks.config.ts" highlight="11"
 import { createClient } from "@liveblocks/client";
 import { createRoomContext } from "@liveblocks/react";
+import { Presence, UserMeta, Storage } from "./types";
```

Comment:
"./types" does not exist here and we didn't introduce `Presence`, `UserMeta` and `Storage` yet. These should come in a future step.